azure-pipelines-woa.yml

```diff
@@ -95,3 +95,19 @@ steps:
     Remove-Item -path $env:APPDATA/Electron* -Recurse
   displayName: 'Delete user app data directories'
   condition: always()
+
+- powershell: |
+    Remove-Item -path $env:APPDATA/app-path -Recurse
+    Remove-Item -path $env:APPDATA/command-line -Recurse    
+    Remove-Item -path $env:APPDATA/crash -Recurse
+    Remove-Item -path $env:APPDATA/default-menu -Recurse
+    Remove-Item -path $env:APPDATA/document-idle -Recurse
+    Remove-Item -path $env:APPDATA/foo -Recurse
+    Remove-Item -path $env:APPDATA/initial-app -Recurse
+    Remove-Item -path $env:APPDATA/locale-check -Recurse
+    Remove-Item -path $env:APPDATA/snapshot-items-available -Recurse    
+    Remove-Item -path $env:APPDATA/site-instance-overrides -Recurse
+    Remove-Item -path $env:APPDATA/remote-control -Recurse
+    Remove-Item -path $env:APPDATA/window-all-closed -Recurse
```

Comment:
oof. new tests that do this stuff aren't going to remember to add to this list.

can we change the tests to all put their data in a specific top-level dir and then delete that? eg. %APPDATA%\electron-tests\[stuff]?

Most tests that do this sort of thing do it by copy-pasting another test, so it'd likely stay good.

Reply:
sure.  we could do that